core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -674,6 +677,18 @@ private[spark] class Executor(
     }
   }
 
+  private def setMDCForTask(taskDescription: TaskDescription): Unit = {
+    val properties = taskDescription.properties
+
+    org.slf4j.MDC.put("appId", properties.getProperty("spark.app.id"))
+    org.slf4j.MDC.put("appName", properties.getProperty("spark.app.name"))
+
+    properties.asScala.filter(_._1.startsWith("mdc.")).foreach { item =>
```

Comment:
This behavior should be documented.

Reply:
Sure, where to add it?

Reply:
How about "Configuring Logging" in docs/configuration.md?

Reply:
Done